app/facades/geocoder_facade.rb

```diff
@@ -2,9 +2,29 @@ class GeocoderFacade
   def self.get_coordinates(address, city, state)
     location = "#{address} #{city},#{state}"
     json_response = GeocoderService.get_coordinates(location)
-    lat_long = {
-      lat: json_response[:results][0][:locations][0][:latLng][:lat],
-      lng: json_response[:results][0][:locations][0][:latLng][:lng]
-    }
+    if GeocoderFacade.validate_location?(address, city, state, json_response)
+      lat_long = {
+        lat: json_response[:results][0][:locations][0][:latLng][:lat],
+        lng: json_response[:results][0][:locations][0][:latLng][:lng]
```

Comment:
you might consider using `long` here. slightly more readable and it's only a single letter. your call though.